tutorials/03_thermal_camera.md

```diff
@@ -152,6 +158,33 @@ Most of the code above is for the model - here's the key piece for temperature a
 
 Once the thermal system plugin is specified, all we need to do is include the `<temperature>` element, which specifies the model's temperature in Kelvin.
 
+### Objects with a varying temperature
+
+Most of the time, objects have a varying temperature.
```

Comment:
Maybe it's just me, but the first time I read this I assumed it meant "time-varying". Maybe it's worth clarifying from the first sentence that it's varying across the surface.

Reply:
I tried to clarify the surface assumption in https://github.com/ignitionrobotics/ign-sensors/pull/79/commits/e888166b05cb0301b7933eb8d7458f36e2262f6c